src/ui/base/text/text.tsx

```diff
@@ -4,7 +4,7 @@ import styles from './text.module.css';
 
 type TextStyle = 'regular' | 'title' | 'monospace' | 'code';
 type TextWeight = 'regular' | 'semibold' | 'bold' | 'extrabold' | 'black';
-type TextColor = 'black' | 'red' | 'white' | 'grey' | 'purple';
+type TextColor = 'black' | 'red' | 'white' | 'grey' | 'purple' | 'auto';
```

Comment:
is there anything using `'black'` right now that would need to be updated to `'auto'`? and can `'black'` just be removed now?

Reply:
I added it at first and then kept it, although black and auto are not being used I thing may be useful to leave it for future updates. Auto is the default contrast color of the light/dark theme, and black and white can be used to force color on specific ui elements that require it

Reply:
yep, what I'm saying is that the current `'black'` prop might be currently used somewhere with the assumption that the site is in light mode - those would need to be changed to auto now, if they exist.

if no usage of the black prop exists, then we should just remove it, since they most likely should be using auto instead, I think